docs/reference/cli-reference/cli-ref-pack.md

```diff
@@ -30,6 +30,7 @@ where `<nuspecPath>` and `<projectPath>` specify the `.nuspec` or project file,
 | --- | --- |
 | BasePath | Sets the base path of the files defined in the [.nuspec](../nuspec.md) file. |
 | Build | Specifies that the project should be built before building the package. |
+| Deterministic | Specify if the command should create a deterministic package. Multiple invocations of the pack command will generate the exact same package. The output of pack is not affected by the ambient state of the machine. Specifically, package internal psmdcp will be generated based on the package files, and the zip entries will be timestamped as 1/1/1980. |
```

Comment:
I feel like the `1/1/1980` date should be ISO 8601 style `1980-01-01`. Given that both the day of month and month of year are the same, there's little room for cultural misunderstanding in this case, but I still think it's best to use an international standard.

If the day and month were not the same, it would be critical to use something else, or at least write the month out as letters instead of a number, as https://docs.microsoft.com/en-us/nuget/ has the same content as https://docs.microsoft.com/en-gb/nuget/, but the second URL gives the impression that it's been localised (with an 's' 😄), whereas in fact it has not.

Reply:
Fixed.